templates/iyo/iyo.py

```diff
@@ -0,0 +1,22 @@
+from js9 import j
+from zerorobot.template.base import TemplateBase
+
+
+class Iyo(TemplateBase):
+
+    version = '0.0.1'
+    template_name = "iyo"
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+
```

Comment:
The template should actually create the  `j.clients.itsyouonline`  profile, hence it should have a client_id and client_secret in its schema

then later on, u do `j.clients.itsyouonline.get(self.name)` to retrieve it

Reply:
I think we need to loose the IYO template. We are not managing IYO here.

What we need to do is that when the JWT is expired, the OVC client should refresh it.

Did that as well for the zero-os client: https://github.com/Jumpscale/lib9/commit/f4825b69a5ddc04f3a87e02e1d604ccd573fb832